lambdas/ingest-upload-file-lambda/src/main/java/uk/gov/ukho/ais/ingestuploadfile/s3/S3ObjectCopier.java

```diff
@@ -25,12 +25,13 @@ public S3ObjectCopier(
 
   public boolean copyS3ObjectToNewLocation(final S3Object s3Object) {
     try {
+      LOGGER.info("Copying {} to {}", s3Object.toString(), destinationBucket);
```

Comment:
Shouldn't need an explicit `toString()`
```suggestion
      LOGGER.info("Copying {} to {}", s3Object, destinationBucket);
```

Reply:
done